_sass/uno.scss

```diff
@@ -1190,27 +1193,33 @@ hr {
 	vertical-align: middle; }
 
 .panel-title {
-	margin: 0 0 5px 0;
```

Comment:
Kinda liked a bit of spacing between these

Reply:
Looks goofy